src/data/moves/move.ts

```diff
@@ -1306,15 +1307,13 @@ export class MoveEffectAttr extends MoveAttr {
    * @param args Set of unique arguments needed by this attribute
    * @returns true if basic application of the ability attribute should be possible
    */
-  canApply(user: Pokemon, target: Pokemon, move: Move, args?: any[]) {
-    return !! (this.selfTarget ? user.hp && !user.getTag(BattlerTagType.FRENZY) : target.hp)
-           && (this.selfTarget || !target.getTag(BattlerTagType.PROTECTED) ||
-                move.doesFlagEffectApply({ flag: MoveFlags.IGNORE_PROTECT, user, target }));
+  canApply(user: Pokemon, target: Pokemon, move: Move, _args?: any[]) {
+    return (this.selfTarget ? user : target).hp > 0
   }
 
   /** Applies move effects so long as they are able based on {@linkcode canApply} */
-  apply(user: Pokemon, target: Pokemon, move: Move, args?: any[]): boolean {
-    return this.canApply(user, target, move, args);
+  apply(user: Pokemon, target: Pokemon, move: Move, _args?: any[]): boolean {
+    return this.canApply(user, target, move, _args);
```

Comment:
```suggestion
  apply(user: Pokemon, target: Pokemon, move: Move, args?: any[]): boolean {
    return this.canApply(user, target, move, args);
```
There's no need to change this.